src/utils/components/ExportModal/constants.ts

```diff
@@ -92,4 +92,4 @@ export const ALREADY_CREATED_ERROR_CODE = 409;
 
 export const UPSTREAM_UPLOADER_IMAGE = 'quay.io/kubevirt/tekton-tasks:v0.23.0';
 export const DOWNSTREAM_UPLOADER_IMAGE =
-  'registry.redhat.io/container-native-virtualization/kubevirt-tekton-tasks-create-datavolume-rhel9:v4.17';
+  'registry-proxy.engineering.redhat.com/rh-osbs/container-native-virtualization-kubevirt-tekton-tasks-create-datavolume-rhel9:v4.18.0-27';
```

Comment:
What is the reason of lock to a fixed build instead of using the flating tag `v4.18` ?

Reply:
I'd prefer to stick to a specific image build that will work, instead of allowing v4.18 that will pull any build image that may be broken.

Reply:
- How can we know, if a specific build will be published at all?

- An externally available image has to be used